packages/myst-frontmatter/src/frontmatter/types.ts

```diff
@@ -24,6 +25,44 @@ export type Biblio = {
   last_page?: string | number;
 };
 
+export type Thebe = {
+  useBinder?: boolean;
+  useJupyterLite?: boolean;
+  requestKernel?: boolean;
+  binderOptions?: ThebeBinderOptions;
+  serverSettings?: ThebeServerSettings;
+  kernelOptions?: ThebeKernelOptions;
+  savedSessionOptions?: ThebeSavedSessionOptions;
+  mathjaxConfig?: string;
+  mathjaxUrl?: string;
+};
+
+export type ThebeBinderOptions = {
+  binderUrl?: string;
+  ref?: string;
```

Comment:
This is the `branch`? Probably leave it matching the thebe options, but it's not clear what this is without looking at the docs.

Reply:
can be any git `ref` including `HEAD`, which also includes tags & commits?
in terms of naming, it pulls directly from the language binder uses so you need to be familar with that, but I think that's ok
![image](https://user-images.githubusercontent.com/1473646/228501603-59dc77d2-50ad-430d-974a-d2bb5decb472.png)